packages/notebook-extension/src/index.ts

```diff
@@ -1459,7 +1463,18 @@ function addCommands(
       const current = getCurrent(args);
 
       if (current) {
-        return NotebookActions.extendSelectionAbove(current.content);
+        return NotebookActions.extendSelectionAbove(current.content, false);
```

Comment:
```suggestion
        return NotebookActions.extendSelectionAbove(current.content);
```

Reply:
I have updated the files with the above changes.